### DISCUSSION TITLE
【委托】昨天被几个老B崽子霸凌了

### DISCUSSION THREAD

**Discussion by SZI726** [share121/inter-knot#1206] [其他]
![8016a9c8ec18d8d3a17a80d35769d3e2](https://github.com/user-attachments/assets/5e541b8b-cbd0-437d-bf04-534a9246e2eb)
就图上那几个，击败他们，公园里的摇摇乐就让你了！

**Comment by 10kaihua** [comment]
我是仙舟人，这就是云上五骁👌

**Reply by Kuuga2000skills** [reply]
其实是年上五骁

**Comment by SZI726** [comment]
昨天和好兄弟们正开开心心玩着摇摇乐，没想到这几个老B崽子直接把俺们打了一顿，多大仇多大怨嘞，后面越想越气，来这求助

**Comment by dont-continue** [comment]
得了吧，不就几个老登吗？收拾他们有多难？我接了！

**Reply by dont-continue** [reply]
到了看到他们了还在下棋等我好消息！！！

**Reply by dont-continue** [reply]
回来了和他们打了三分钟，他们毫发无伤我倒是差点进医院了，这委托我还是不接了吧？这谁能做？

**Comment by gaojiaobei** [comment]
那天我看到他们在和一个绿色头发小姑娘下棋，那小姑娘看着就好欺负，建议从她下手

**Reply by dont-continue** [reply]
得了吧我上次看到他和治安局的人混在一起，说不定是治安局的人呢？咱还是不要找到不必要的麻烦比较好😟

**Comment by ZhuYuan-148** [comment]
不要去招惹公园里的那些老人哦。主要是因为比起他们，我们还是更担心各位的生命安全。

**Comment by ZhuYuan-148** [comment]
打架赢了坐牢，输了进院。希望大家能口头解决问题。
![1723215836510](https://github.com/user-attachments/assets/742c3473-fe10-4ded-83f4-5b29c6e0d0bd)

**Comment by Lanxey** [comment]
这个委托接不了，这几个老头老太现在认识的人不多，但二十年前提起【光映五骁】谁人不知谁人不晓？据传带着他们的雕像进空洞，以骸碰到都绕路，惹不起，惹不起。

**Comment by GanlinSama** [comment]
几个老头老太太能有什么本事？摔一跤都能骨折怎么还给楼主打了？

**Comment by XiaoFffeng** [comment]
我还没有蠢到和治安局作对，@青衣

**Comment by sfhjhsdfg** [comment]
中间那个看起来是治安局的装束，你这是犯了事吧

**Comment by zzziloveyou** [comment]
好像你被打的照片还被发到了治安局墙上。。。

**Comment by tuzibuqiahuluobo** [comment]
这委托， 狡兔屋都不敢接吧！

**Comment by gaojiaobei** [comment]
治安局的来了，我先跑了

**Comment by fu-fu404** [comment]
我是天理，这打不过

**Comment by qwq1949** [comment]
这你得找治安局的来

**Comment by uyfh352** [comment]
這委託不能亂接，~~搞不好就進局子了~~
